frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -114,17 +94,28 @@ function EmbedFrame({
   actionButtons,
   dashboardTabs = null,
   footerVariant = "default",
-  location,
-  hasEmbedBranding,
   parameters,
   parameterValues,
   draftParameterValues,
   hiddenParameterSlugs,
   setParameterValue,
   setParameterValueToDefault,
   enableParameterRequiredBehavior,
-}: Props) {
-  const [hasFrameScroll, setHasFrameScroll] = useState(true);
+  bordered,
+  titled,
+  theme,
+  hide_parameters,
+  hide_download_button,
+}: EmbedFrameProps) {
+  const isSdk = useSelector(getIsEmbeddingSdk);
```

Comment:
At some point I'd like to remove this check and either pass a `ParametersList` or `SyncedParametersList` component through the props or something